airflow-core/tests/unit/models/test_mappedoperator.py

```diff
@@ -1401,3 +1405,60 @@ def t3(a):
     dr.task_instance_scheduling_decisions()
     ti3 = dr.get_task_instance(task_id="tg1.t3")
     assert not ti3.state
+
+
+def test_properties():
+    op = PythonOperator.partial(
+        task_id="mapped",
+        python_callable=print,
+        email="email",
+        execution_timeout=timedelta(seconds=10),
+        retry_delay=timedelta(seconds=5),
+        max_retry_delay=timedelta(seconds=60),
+        retry_exponential_backoff=True,
+        max_active_tis_per_dag=1,
+        max_active_tis_per_dagrun=2,
+        run_as_user="user",
```

Comment:
We should probably use `pytest.mark.parametrize` to test more cases on some attributes (default vs user-provided)